scss/components/_menu.scss

```diff
@@ -0,0 +1,51 @@
+.menu {
+  display: flex;
+
+  align-items: center;
+  &__item {
+    margin-left: 28px;
+  }
+  &__link {
+    display: block;
+    font-size: 14px;
+    font-weight: 700;
+    color: $primary;
+    text-transform: uppercase;
+    text-decoration: none;
+    letter-spacing: 1px;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
+  @media only screen and (max-width: $md) {
+    &__link {
+      font-size: 12px;
+    }
+  }
+  @media only screen and (max-width: $xs) {
+    &__link {
+      font-size: 11px;
+    }
+  }
+}
+
+.hamburger {
```

Comment:
why 2 blocks in one file? Please extract `hamburger ` into separate file or make it an element of menu block